elmslie-android/src/main/java/vivid/money/elmslie/android/base/ElmActivity.kt

```diff
@@ -2,26 +2,40 @@ package vivid.money.elmslie.android.base
 
 import androidx.annotation.LayoutRes
 import androidx.appcompat.app.AppCompatActivity
-import vivid.money.elmslie.android.screen.ElmDelegate
-import vivid.money.elmslie.android.screen.ElmScreen
-import vivid.money.elmslie.android.storeholder.LifecycleAwareStoreHolder
-import vivid.money.elmslie.android.storeholder.StoreHolder
-import vivid.money.elmslie.android.util.fastLazy
+import androidx.lifecycle.SavedStateHandle
+import vivid.money.elmslie.android.elmStore
+import vivid.money.elmslie.android.renderer.ElmRenderer
+import vivid.money.elmslie.android.renderer.ElmRendererDelegate
+import vivid.money.elmslie.android.storestarter.ViewBasedStoreStarter
+import vivid.money.elmslie.core.store.Store
 
+@Deprecated("Please use ElmRenderer and ViewBasedStoreStarter in you base classes. Will be removed in next releases. ")
 abstract class ElmActivity<Event : Any, Effect : Any, State : Any> :
```

Comment:
as we don't use this and elmfragment in our projects — maybe it's time to remove them?  
as in any case we're doing breaking changes here

Reply:
I prefer to remove it in the next releases. With these classes, fixing breaking changes will be a little bit easy. 
@rcmkt what do you think?

Reply:
Agree with Jenya, it's not a stable release, so we can remove it a bit later